public/index.html

```diff
@@ -0,0 +1,370 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>W2-A2-Benton-Mark</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+
+<body>
+    <a id="home"></a>
+    <nav class="drop-down">
+        <ul class="flex-container nav-ul">
+            <li class="selected nav-li"><a href="#home">Home.</a>
+            <li class="hidden-class nav-li"><a href="#services">Services.</a></li>
+            <li class="hidden-class nav-li"><a href="#portfolio">Portfolio.</a></li>
+            <li class="hidden-class nav-li"><a href="#about">About.</a></li>
+            <li class="hidden-class nav-li"><a href="#team">Team.</a></li>
+            <li class="hidden-class nav-li"><a href="#contact">Contact.</a></li>
+        </ul>
+    </nav>
+    <section class="header-section">
+        <header>
+            <img src="./imgs/top-logo.gif" alt="main page logo">
+            <p>Hi there! We're Avengers, <strong>creative design agency</strong>
+                <br>we build <span class="awesome-sm"><strong>awesome</strong></span> mobile and websites</p>
+            <object class="awesome" type="image/svg+xml" data="./imgs/underline.svg"></object>
+        </header>
+    </section>
+    <section class="section-orange">
+        <object class="clip clip-white" type="image/svg+xml" data="./imgs/clipwhite.svg">
+        </object>
+        <a id="services"></a>
+        <h1>SERVICES</h1>
+        <p>Contrary to popular belief, Lorem Ipsum is not simply random. It has roots in a piece of classical Latin literature from 45BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia</p>
+        <div class="column-two">
+            <p class="column-left">Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industry standard dummy text since the 1500's, when an unknown printer took a galley of type and scrambled it to make a type specimen book</p>
+            <p class="column-right">Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industry standard dummy text since the 1500's, when an unknown printer took a galley of type and scrambled it to make a type specimen book</p>
+        </div>
+        <ul class="flex-container">
+            <li class="flex-item-1">
+                <h2>UI &#38; UX Design</h2>
+                <img src="./imgs/brush.gif" alt="brush logo">
```

Comment:
There were a number of ways you could have gone with this; `img` is valid, but I would like to have seen something more advanced. If you wanted to stick with the bitmap, you could do a background image either on the `h2` itself or on pseudo-content after the `h2`. An icon font would also have been good. And SVG (inline or background-image—_not_ `object`) would have been best.

The bitmap as submitted is low resolution and looks very blocky on my retina MacBook. Not ideal.